Packages/com.unity.inputsystem/InputSystem/AssemblyInfo.cs

```diff
@@ -7,6 +7,7 @@
 [assembly: InternalsVisibleTo("Unity.InputSystem.Tests.Editor")]
 [assembly: InternalsVisibleTo("Unity.InputSystem.Tests")]
 [assembly: InternalsVisibleTo("Unity.InputSystem.IntegrationTests")]
+[assembly: InternalsVisibleTo("Unity.InputSystem.ForUI")] // To avoid minor bump
```

Comment:
Note that this is to avoid a minor version bump. Ideally, if we have some verifier plugin functionality we should consider making it public in the future if plugins are truly supported. At this point I do not think the API is capable enough or mature enough for such exposure.